zpls_echograms/README.md

```diff
@@ -1,104 +1,193 @@
+data directory would be `./CE04/PC01B/ZPLSCB102_10.33.10.143/2017`. 
+
+Processed data is created in a folder of the users choice. We recommend placing the data at the same level as the root 
+data directory in a separate folder called `processed`. The data in the `processed` folder is organized into subfolders 
+named according to the date range entered for processing. For example, using the AZFP data from above, if the date range
+was 2017-01-01 through 2017-01-08, then the processed data would end up in a folder `20170101-20170108`. The full path
+would be `./GA/ZPLSG_sn55067/processed/20170101-20170108/`
+
+There are three types of processed data created by the zpls_echogram script:
+
+1. **Converted**: These are files converted from the raw format (either `.01A` or `.raw`) to NetCDF format (`.nc`) by 
+   the echopype convert function. They are 1:1 with the source raw data file. These `.nc` files use the 
+   [SONAR-NetCDF4](http://www.ices.dk/sites/pub/Publication%20Reports/Cooperative%20Research%20Report%20(CRR)/CRR341.pdf)
+   formatting convention.
+2. **Processed**: The echopype process function, taking the converted files as inputs, is used to calculate the volume 
+   acoustic backscatter strength (Sv), and the range (distance from the sensor face) in meters. This subset of data is 
+   saved in daily `.nc` files that use the site and date range to construct the file name 
+   (e.g. data for 2016-01-07 would be saved in 
+   `GA02HYPM_Upper_Bioacoustic_Echogram_20170101-20170108_Calibrated_Sv_Full_20170107.nc`).
+3. **Averaged**: Takes the processed data one step further and applies temporal averaging (15 minute median averages for
+   the coastal ZPLSC and 60 minute median averages for the global ZPLSG) for use in creating the echograms. Missing data
+   is filled in via linear interpolation for gaps less than 3x the averaging time window. The averaged data is saved 
+   in a `.nc` file using a similar file naming convention as the processed data above (e.g. 
+   `GA02HYPM_Upper_Bioacoustic_Echogram_20170101-20170108_Calibrated_Sv_Averaged.nc`).
+   
+An echogram covering the time period specified by the date range is created from the averaged data and saved along with
+the three data types list above. The files share the same naming convention as the processed and averaged data (e.g. 
+`GA02HYPM_Upper_Bioacoustic_Echogram_20170101-20170108_Calibrated_Sv.png`).
+
+**Processing Examples**
+
+The examples below show how to use the script with the different options.
+
+**Example 1:**
+```
+# set default data directories and .xml file 
+GA_DATA="./GA/ZPLSG_sn55067/DATA"
+GA_PROC="./GA/ZPLSG_sn55067/processed"
+GA_XML="./GA/ZPLSG_sn55067/DATA/201610/16101417.XML"
+
+# set the site, the script will look at the mooring configuration and apply the 
+# tilt angle correction (15 degrees for GA02HYPM_Upper) and the theoretical 
```

Comment:
Perhaps "design depth" would be better than "theoretical depth"

Reply:
Agreed, made the change here and further down in Example 4.